### PR TITLE
Replace {{event)}} with domxref (batch #2)

### PR DESCRIPTION
**Pull request by teoli2003**
Now that we have updated events on `MDN/content`, we can replace `{{event}}` (that is ambiguous or lead to unwanted redirects) with the `{{domxref}}` macros.

### CODE REVIEW

File: files/en-us/web/api/paymentresponse/payerdetailchange_event/index.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ Although this event type is {{domxref("PaymentRequestUpdateEvent")}}, it doesn't
 
 ## Examples
 
-In the example below, `onpayerdetailchange` is used to set up a listener for the {{event("payerdetailchange")}} event in order to validate the information entered by the user, requesting that any mistakes be corrected
+In the example below, `onpayerdetailchange` is used to set up a listener for the {{domxref("PaymentRequest.payerdetailchange_event", "payerdetailchange")}} event in order to validate the information entered by the user, requesting that any mistakes be corrected
```
Comment (hamishwillee) [inline files/en-us/web/api/paymentresponse/payerdetailchange_event/index.md:47]: Is `PaymentRequest.payerdetailchange_event` correct? It's giving a macro error and not listed in the parent page or BCD for `PaymentRequest`.

**Review by hamishwillee — APPROVED** [review]
Looks great. Approving, but not merging to give you a chance to look at https://github.com/mdn/content/pull/14578#discussion_r841300609

And of course resolving the conflicts :-(

File: files/en-us/web/api/paymentresponse/payerdetailchange_event/index.md (hunk 1 above)
Comment (teoli2003) [reply files/en-us/web/api/paymentresponse/payerdetailchange_event/index.md:47]: Good catch, it is `PaymentResponse`

**Comment by teoli2003** [comment]
Thanks: I fixed the two conflicts, and did the fix for the broken link.

**Pull request merged**